### PR TITLE
Mark "idle" to translation

### PR DESCRIPTION
**Pull request by Faalagorn**
#### Summary
None

#### Purpose of change
Add "idle" string to translations, which can appear in side menu widgets for AI needs (goals).
screenshot_2022-01-29-183416

#### Describe the solution
Surround appropriate functions with `_( )` function 

#### Describe alternatives you've considered
There are more untranslated strings in this file, so I was thinking if they should also be marked for translation, but I don't know where they appear. Feel free to add them too, if needed.
 

#### Testing
Compiled the game, added msgid "idle?" and translation for the message, run compile_mo.sh and see if the string appears in game (it does).

### CODE REVIEW

File: src/behavior.cpp [hunk 1]
```diff
@@ -11,6 +11,7 @@
 #include "generic_factory.h"
 #include "debug.h"
 #include "json.h"
+#include "translations.h"
```
Comment (Faalagorn) [inline src/behavior.cpp:14]: I'm not sure if it's needed? Can someone check?

**Comment by BrettDong** [comment]
I believe it is behavior tree internal state data and overwriting that string with translated content could break AI behavior.

**Comment by Faalagorn** [comment]
> I believe it is behavior tree internal state data and overwriting that string with translated content could break AI behavior.

Thanks for pointing that out; can you verify that with that PR? If that is indeed the case, maybe there's a more clever idea to wrap that string around some other string which then can be translated?

**Comment by kevingranade** [comment]
This is debug output, which we don't translate.

**Comment by Faalagorn** [comment]
> This is debug output, which we don't translate.

Thanks! Guess a more sophisticated way should be found for this string, probably some `if` to display an actual string instead of just printing "idle" to the sidebar, something to take a look that might go well with @wapcaplet's sidebar revamp, even in English :)

**Pull request closed without merging**